components/color-picker/util.ts

```diff
@@ -18,3 +18,5 @@ export const genAlphaColor = (color: Color, alpha?: number) => {
   hsba.a = alpha || 1;
   return generateColor(hsba);
 };
+
+export const isColorCleared = (color: Color) => color.equals('#00000000');
```

Comment:
这样不好，期望是清楚颜色之后还原 alpha 还是原来的 hsv 值，这样就变成黑色了

Reply:
清除颜色为啥还要还原？

Reply:
清楚颜色不改变 hsb 数值，这样面板颜色就不会变

Reply:
I also think that the color values should be reset when it is cleared.
This is also question about the `clear` action of the `color picker`.
a 0% opacity of the color '#ff0000' means that the color '#ff0000' but with an opacity of 0. not `clear`.
I Think if the color picker is cleared, it should literally have no color value. It could become "" or null, or maybe undefined.